PREREQUISITES.md

```diff
@@ -360,7 +360,7 @@ problems.
 - Download the latest SGX SSL git repository for your version of OpenSSL:
```

Comment:
Download the latest SGX SSL git repository -> Download specific version of SGX SSL git repository.  Using specific SGX SSL tag "lin_2.5_1.1.1d" corresponding to openSSL version 1.1.1d